deploy/operator/overlays/dev-template/kustomization.yaml

```diff
@@ -12,8 +12,17 @@ images:
   newName: quay.io/${JBS_QUAY_IMAGE}/hacbs-jvm-controller
   newTag: dev
 
-
 patches:
+- patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/env
+      value:
+        - name: S3_SYNC_ENABLED
+          value: |
+            ${JBS_S3_SYNC_ENABLED}
+  target:
+    kind: Deployment
+    name: hacbs-jvm-operator
```

Comment:
Moved the patch here so all operator patches are together (such as the imagePullPolicy below)